redbot/core/utils/__init__.py

```diff
@@ -498,3 +508,81 @@ def map(self, func: Callable[[_T], Union[_S, Awaitable[_S]]]) -> AsyncIter[_S]:
             raise TypeError("Mapping must be a callable.")
         self._map = func
         return self
+
+
+def get_end_user_statement(file: Union[Path, str]) -> Optional[str]:
+    """
+    This function attempts to get the ``end_user_data_statement`` key from a cogs ``info.json``. This will log the reason if ``None`` is returned.
+
+    Parameters
+    ----------
+    file: Union[pathlib.Path, str]
+        The ``__file__`` variable for the cogs ``__init__.py`` file.
+
+    Returns
+    -------
+    Optional[str]
+        The end user statement found in the info.json or ``None`` if there was an issue finding one.
+
+    Examples
+    --------
+    >>> # In cogs `__init__.py`
+    >>> from redbot.core.utils import get_end_user_statement
+    >>> __red_end_user_data_statement__  = get_end_user_statement(__file__)
+    >>> def setup(bot):
+            ...
```

Comment:
Python's REPL uses `... ` prompt for continuation
```suggestion
    >>> def setup(bot):
    ...     ...
```